daemon/algod/api/algod.oas2.json

```
@@ -3880,6 +3884,23 @@
         }
       }
     },
+    "ScratchSlotWrite": {
+      "description": "The write operation into a scratch slot.",
+      "type": "object",
+      "required": [
+        "scratch-slot-id",
```

I'd prefer the simplicity of "slot". Since it's a field in "ScratchSlotWrite"
```suggestion
        "slot",
```

---

I also think "new-value" could just be "value", since the "new" is implied by it being a "write".